### ISSUE TITLE
[ace] vcpkg install ace:x64-windows fails on Visual Studio 2019 16.5

### ISSUE THREAD

**Issue by traversaro** [microsoft/vcpkg#10522]
**Host Environment**
- OS: Windows
- Compiler: Visual Studio 2019 16.5

**To Reproduce**
`./vcpkg install ace:x64-windows`

**Failure logs** 
See https://developercommunity.visualstudio.com/content/problem/954522/internal-compiler-error-1925286104.html . 

**Additional context**
This is  not directly a vcpkg issue, but I recently discovered it, and as it will  be only fixed with Visual Studio 2019 16.6 (that I guess it will not  come out soon) I thought it was better to open a dedicate issue to raise awareness about this.

More details: 
* https://github.com/actions/virtual-environments/issues/605
* https://developercommunity.visualstudio.com/content/problem/954522/internal-compiler-error-1925286104.html

**Comment by LilyWangL** [comment]
Thanks for your issue, it's very useful to other users who need install ace on Visual Studio 16.5.

**Comment by jwillemsen** [comment]
See also https://github.com/DOCGroup/ACE_TAO/issues/1059

**Comment by timautry** [comment]
Applies to VS 2019 16.5.2 also :(

**Comment by SkyFire** [comment]
Still broke in the latest VS release. Micro$oft needs to get this fixed. I compiles in the preview version.
https://visualstudio.microsoft.com/vs/preview/

**Comment by jwillemsen** [comment]
See https://github.com/microsoft/vcpkg/pull/11369

**Comment by jwillemsen** [comment]
Change has been merged into ACE, see https://github.com/DOCGroup/ACE_TAO/pull/1114

**Comment by timautry** [comment]
Even after #11369 I am still getting an INTERNAL COMPILER ERROR... 
I even deleted my vcpkg vs2019 instance and recreated from scratch:
`     2>D:\Git\vs2019.prod\buildtrees\ace\src\ACE-src-6-bbdc7cf805\ace\Process_Manager.cpp(144,1): fatal error C1001: Internal compiler error. [D:\Git\vs2019.prod\buildtrees\ace\src\ACE-src-6-bbdc7cf805\ace\ACE.vcxproj]`
`         (compiler file 'msc1.cpp', line 1532)`
`          To work around this problem, try simplifying or changing the program near the locations listed above.`
`         If possible please provide a repro here: https://developercommunity.visualstudio.com`
`         Please choose the Technical Support command on the Visual C++`
`         Help menu, or open the Technical Support help file for more information`
`         INTERNAL COMPILER ERROR in 'C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.25.28610\bin\HostX64\x64\CL.exe`
`             Please choose the Technical Support command on the Visual C++`
`             Help menu, or open the Technical Support help file for more information`
`     2>Done Building Project "D:\Git\vs2019.prod\buildtrees\ace\src\ACE-src-6-bbdc7cf805\ace\ACE.vcxproj" (Rebuild target(s)) -- FAILED.`
`     1>Done Building Project "D:\Git\vs2019.prod\buildtrees\ace\src\ACE-src-6-bbdc7cf805\ace\ace.sln" (Rebuild target(s)) -- FAILED.`

Attached log:
[build-x64-windows-rel-out.log](https://github.com/microsoft/vcpkg/files/4647284/build-x64-windows-rel-out.log)

**Comment by jwillemsen** [comment]
This PR has first to be merged, the fix is in ace but there is no new ace release yet

**Comment by timautry** [comment]
I apologize. I am still new at the git workings. I thought it had been merged, but I am mistaken.

**Comment by timautry** [comment]
FYI @jwillemsen @traversaro I was able to compile the current ace v6.5.9 with the newly released VS 2019 16.6...

**Comment by jwillemsen** [comment]
This should been fixed, @traversaro can you retest and close this issue when it works for you?

**Comment by traversaro** [comment]
> 
> 
> This should been fixed, @traversaro can you retest and close this issue when it works for you?

Thanks a lot @jwillemsen , I checked in https://github.com/traversaro/reproduce-ace-vcpkg-github-actions-fails/pull/2 and indeed this solves the problem, thanks a lot!
